app/main.py

```diff
@@ -2,0 +1,22 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    not_vaccinated = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            not_vaccinated += 1
+        except NotWearingMaskError:
+            masks_to_buy += 1
+
+    if not_vaccinated:
+        return "All friends should be vaccinated"
```

Comment:
The message 'All friends should be vaccinated' might be misleading if only some friends are not vaccinated. Consider rephrasing it to indicate the actual number of friends who are not vaccinated.